src/app/[locale]/(home)/products/[productId]/_components/add-to-cart-button.tsx

```
@@ -1,13 +1,14 @@
 "use client";
 import { Button } from "@components/ui/button";
 import { Check, ShoppingCart } from "lucide-react";
-import React, { useEffect, useState } from "react";
+import React, { use, useEffect, useState } from "react";
```

ESLint

[error] 'use' is defined but never used.

(@typescript-eslint/no-unused-vars)